### DISCUSSION TITLE
pyspark integration - walkaround for explode

### DISCUSSION THREAD

**Discussion by l0kr** [duckdb/duckdb#12074] [Q&A]
I am testing my pyspark codebase with duckdb, but explode function is missing from `duckdb.experimental.spark.sql.functions`
Is there any workaround for this?

**Comment by Tishj** [comment]
It sounds like it's similar to UNNEST?

**Comment by soerenwolfers** [comment]
Unnest doesn't work on maps in duckdb, so you'll have to combine it with map_keys and map_values as demonstrated at https://github.com/duckdb/duckdb/discussions/11851

**Comment by l0kr** [comment]
This means I can use custom duckdb operation on dataframe created by SparkSession from  `duckdb.experimental.spark.sql`? 
How does it look like?
